driver/ni/analog_read_test.cpp

```diff
@@ -0,0 +1,1226 @@
+// Copyright 2024 Synnax Labs, Inc.
+//
+// Use of this software is governed by the Business Source License included in the file
+// licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with the Business Source
+// License, use of this software will be governed by the Apache License, Version 2.0,
+// included in the file licenses/APL.txt.
+
+#include <include/gtest/gtest.h>
```

Comment:
File still needs removal of comments, a lot of formatting changes, and removal of extra whitespace